vowpalwabbit/core/src/reductions/metrics.cc

```diff
@@ -136,6 +136,12 @@ void predict_or_learn(metrics_data& data, T& base, E& ec)
 
 void additional_metrics(VW::workspace& all, VW::metric_sink& sink)
 {
+  sink.set_float("sum_loss", all.sd->sum_loss);
+  sink.set_float("weighted_labeled_examples", all.sd->weighted_labeled_examples);
```

Comment:
Weighted examples is already exposed so it feels like this should just be added next to it

[double get_weighted_examples(vw_ptr vw) { return vw->sd->weighted_examples(); }](https://vowpalwabbit.org/docs/vowpal_wabbit/python/latest/reference/vowpalwabbit.pyvw.pylibvw.html#vowpalwabbit.pyvw.pylibvw.vw.get_weighted_examples)

Reply:
will remove